h_transport_materials/materials/lipb.py

```diff
@@ -0,0 +1,245 @@
+    source=fauvet_src,
+    name="H Fauvet (1988)",
+    author="fauvet",
+    year=1988,
+    isotope="H",
+)
+fauvet_solubility = Solubility(
+    pre_exp=2.7e-08 * atom_density_lipb,
+    act_energy=0,
+    range=(722, 724),  # TODO should be 723 link to issue #37
+    source=fauvet_src,
+    name="H Fauvet (1988)",
+    author="fauvet",
+    year=1988,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+
+
+schumacher_src = "R. Schumacher, A. Weiss, DOI:10.1002/bbpc.19900940612"
+schumacher_solubility = Solubility(
+    pre_exp=8.98e-07 * atom_density_lipb,
+    act_energy=6100 * k_B / Rg,
+    range=(508, 1040),
+    source=schumacher_src,
+    name="H Schumacher (1990)",
+    author="schumacher",
+    year=1990,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+
+
+reiter_src = "F. Reiter, DOI:10.1016/0920-3796(91)90003-9"
+reiter_diffusivity_h = ArrheniusProperty(
+    pre_exp=4.03e-08 * atom_density_lipb,
+    act_energy=19500 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="H Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="H",
+)
+reiter_diffusivity_d = ArrheniusProperty(
+    pre_exp=4.03e-08 * atom_density_lipb,
+    act_energy=19500 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="D Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="D",
+)
+reiter_diffusivity_t = ArrheniusProperty(
+    pre_exp=4.03e-08 * atom_density_lipb,
+    act_energy=19500 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="T Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="T",
+)
+reiter_solubility_h = Solubility(
+    pre_exp=2.44e-08 * atom_density_lipb,
+    act_energy=1350 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="H Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+reiter_solubility_d = Solubility(
+    pre_exp=2.36e-08 * atom_density_lipb,
+    act_energy=1350 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="D Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="D",
+    units="m-3 Pa-1/2",
+)
+reiter_solubility_t = Solubility(
+    pre_exp=2.32e-08 * atom_density_lipb,
+    act_energy=1350 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="T Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="T",
+    units="m-3 Pa-1/2",
+)
+
+
+aiello_src = (
+    "A. Aiello, A. Ciampichetti, G. Benamati, DOI:10.1016/j.fusengdes.2005.06.364"
+)
+aiello_solubility = Solubility(
+    pre_exp=4.66e-06 * atom_density_lipb,
+    act_energy=13399 * k_B / Rg,
+    range=(600, 900),
+    source=aiello_src,
+    name="H Aiello (2006)",
+    author="aiello",
+    year=2006,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+
+
+shibuya_src = (
+    "Y. Shibuya, M. Aida, Y.Fujii, M. Okamoto, DOI:10.1016/0022-3115(87)90006-7"
+)
+shibuya_diffusivity = ArrheniusProperty(
+    pre_exp=2.62e-03 * atom_density_lipb,
```

Comment:
Shibuya 1987 used Li17-Pb83

Reply:
![image](https://user-images.githubusercontent.com/40028739/171630343-5b26f3fc-48f7-4513-ba52-aff9b84dd866.png)
Shibuya doesn't provide fitted values. Only experimental points so we should refit the data ourselves

Reply:
Btw, the values given by shibuya and the values you gave don't seem to be the right order of magnitude. the conversion from cm2 to m2 wasn't right.